### DISCUSSION TITLE
Can I do this in KOReader?

### DISCUSSION THREAD

**Discussion by ghost** [koreader/koreader#11673] [Q&A]
Can I hide by the tap the top bar like the bottom bar? It will be very useful if I can set and hide the top bar only by the tap and not by rummaging through the settings...

**Comment by jonnyl2** [comment]
You can set up two gestures (or a QuickMenu) to turn it on or off, respectively. 

It's in the Gesture Manager under 

`Reflowable documents -> Alt Status Bar ->`
